app.js

```diff
@@ -1,81 +1,132 @@
+    div.className = `alert alert-${className}`;
+    div.appendChild(document.createTextNode(message));
+    const container = document.querySelector('.container');
+    const form = document.querySelector('#book-form');
+    container.insertBefore(div, form);
 
-              `,
-    );
+    // vanish in three seconds
+    setTimeout(() => document.querySelector('.alert').remove(), 3000);
+  }
+
+  static clearFields() {
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
+  }
+
+  //  store class handles storage, store the book name class
+
+  static getBooks() {
+    const books = JSON.parse(localStorage.getItem('books')) || [];
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = this.getBooks();
+    books.push(book);
+    this.saveBooks(books);
+  }
 
-    booksContainer.innerHTML = displayBook.join('');
+  //   Save books to locastorage
+  static saveBooks(books) {
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static count = 0;
+
+  //   Declare unique Id
+  static uniqueId() {
+    /* eslint-disable no-plusplus */
+    const id = this.count++;
+    return `book_${id}`;
+  }
+
+  static removeBook(id) {
+    let books = this.getBooks();
+
+    // Filter books based on the remove button id and return the res
+    books = books.filter((book) => book.id !== id);
+
+    // show success message
+    this.showAlert('Book Removed', 'danger');
+
+    // Then save to local storage again
+    this.saveBooks(books);
+
+    // Then render the remaining books again
+    this.displayBooks();
   }
 }
 
-// Add eventListener to the button, to run each both functions everytime the button is clicked
-AddBookBtn.addEventListener('click', (e) => {
+// event: display books
+window.addEventListener('DOMContentLoaded', Book.displayBooks);
+
+// event: add book
+const addBook = document.querySelector('.addBook');
+
+addBook.addEventListener('click', (e) => {
+  // prevent actual submit
   e.preventDefault();
-  addBooks();
-  saveBooks();
-  renderBooks();
-  clearInput();
-});
 
-/* eslint-disable no-unused-vars */
-function removeBook(id) {
-  bookCollections = bookCollections.filter((book) => book.id !== id);
-  saveBooks();
-  renderBooks();
-}
+  // get form values
+  const title = document.querySelector('#title');
+  const author = document.querySelector('#author');
+  const Id = Book.uniqueId();
+
+  // validate
+  if (title.value === '' || author.value === '') {
+    Book.showAlert('please fill in all feilds', 'danger');
+  } else {
+    //  Instatiate a book
+    const book = new Book(title.value, author.value, Id);
 
-window.addEventListener('DOMContentLoaded', () => {
-  renderBooks();
+    //  Add book to store
+    Book.addBook(book);
+
+    // show success message
+    Book.showAlert('Book Added', 'success');
+
+    // Render books
+    Book.displayBooks();
+
+    //  Clear feilds
+    Book.clearFields();
+  }
 });
```

Comment:
![image](https://github.com/AbuTalha3/awesomeBooksList/assets/93039359/a9c01198-0764-4754-8d3c-579018b7b239)

- [ ] It would be nice if you fix the link highlighted in the image above to make your readme descriptive and professional 👍If you don't have any section for this link then kindly remove this link 😊